src/Contracts/UnsupportedOperationException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace LaraCrafts\UrlShortener\Contracts;
```

Comment:
I am not sure putting an exception in the contracts namespace.

Reply:
Where would you put this then? I kinda just copied this from Laravel hahaha

Reply:
Simply into `LaraCrafts\UrlShortener\Exceptions` namespace.

Reply:
For now I'll leave it, if we need to add any more exceptions, they'll get their own namespace